xenserver/settings.py

```diff
@@ -0,0 +1,157 @@
+import datetime
+import os
+
+import djcelery
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+def abspath(*args):
+    """convert relative paths to absolute paths relative to BASE_DIR"""
+    return os.path.join(BASE_DIR, *args)
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '02qbonw5tuu$m(c^s9h5u+ravid-i+@kn(^5^aw!@o^nl*+!9c'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
It's tricky to change this and the `SECRET_KEY` to be required without making the dev setup more tricky... But I'll see what I can do.